{
  "contest": 427,
  "handle": "VulnViper",
  "risk": "3",
  "title": "Out-of-Bounds Error in getPositionsForCurator Function",
  "issueId": 325,
  "issueUrl": "https://github.com/code-423n4/2024-08-phi-validation/issues/325",
  "validatedIssueId": 215,
  "validatedIssueUrl": "https://github.com/code-423n4/2024-08-phi-findings/issues/215"
}